src/main/scala/com/metl/model/Reputation.scala

```diff
@@ -25,9 +22,9 @@ object Reputation extends Logger{
   def addStanding(who:String,what:GainAction.Value):Unit = Stopwatch.time("Reputation:addStanding(%s,%s)".format(who,what),{
     val score = Informal.value(what)
     if (score > 0){
-      val currentStanding = standingMap(who)
+      val currentStanding = standingMap.getOrElseUpdate(who, Informal.standing(who))
```

Comment:
I thought it was the initial default score, which is then incremented by the newScore.  Trying not to change existing behaviour, as it used to have a Default.